src/api/src/firebase/configs.ts

```diff
@@ -3,7 +3,7 @@ require('dotenv').config();
 let adminCredentials = {}; // firebase-admin service account credentials
 let firebaseCredentials = {}; // firebase SDK configuration
 
-process.env.NODE_ENV === process.env.NODE_ENV || 'development';
+process.env.NODE_ENV = process.env.NODE_ENV || 'development';
```

Comment:
nice

Reply:
yeah, I also fixed that as well since we discovered it in the call